_episodes/06-photo.md

```diff
@@ -101,22 +101,26 @@ filter](https://en.wikipedia.org/wiki/Photometric_system).
 The x-axis shows the difference in apparent magnitude between the g
 and i filters, which indicates color.
 
-Stars with lower values of (g-i) are "bluer" since they are brighter in the g-band than in
-the i-band, compared to other stars. To a first order approximation, the color of a star is 
-related to the star's temperature, with bluer stars indicating higher temperatures and redder stars 
-indicating lower temperatures. An important second order effect involves 
-the metallicity, or the amount of metals (elements heavier than helium, in this case) 
-that are present in a star's atmosphere. 
-Higher metallicity leads to redder stars and lower metallicity leads to bluer stars. 
-
-Stars in the lower-left quadrant of this diagram are faintest and bluest suggesting 
-they are have lower metallicity than
-the other stars, which means they are [likely
-to be
-older](http://spiff.rit.edu/classes/ladder/lectures/ordinary_stars/ordinary.html).
-
-Since we expect the stars in GD-1 to be older than the foreground and background
-stars, and farther away, the stars in the lower-left are more likely to be in GD-1.
+
+> ## A slight detour for non-domain experts
+>
+> If you are wondering how (as a non-domain expert)
+> how to interpret this color-magnitude diagram, please expand the section below.
+> 
+> > ## The color magnitude diagram of GD-1 and foreground stars
+> > 
+> > As a pathologist can easily point to tumor on a biopsy slide, so too can many astronomers see two stellar populations in this color magnitude diagram, one from an old star cluster (presumably, GD-1), and the other, stars much closer, but at every distance between the Earth and the cluster ("foreground"). The color magnitude diagram was developed to separate these features out just as pathologists depend on H&E staining to contrast human tissue.
```

Comment:
I like the analogy, I have a slight concern that this might need a trigger warning for potential cancer discussion. 

Since H&E staining may not be common knowledge and to avoid unnecessary additional definition, I recommend changing last sentence here to: The color magnitude diagram is a technique developed to separate these features out just as pathologist have techniques to contrast human tissue.

Reply:
Accepted.